internal/uvm/create.go

```diff
@@ -315,3 +316,15 @@ func (uvm *UtilityVM) normalizeMemorySize(ctx context.Context, requested int32)
 func (uvm *UtilityVM) DevicesPhysicallyBacked() bool {
 	return uvm.devicesPhysicallyBacked
 }
+
+// Closes the external GCS connection if it is being used and also closes the
+// listener for GCS connection.
+func (uvm *UtilityVM) CloseGCSConnection() (err error) {
```

Comment:
Does this actually need to be exported? Seems like this would only ever be called by the UVM itself/in this package.

Reply:
This needs to be exported because in one of the later PRs we call this function from `cmd/containerd-shim-runhcs-v1/clone.go` to close the GCS connection just before saving the uvm.